.github/workflows/dynamiclinks.yml

```diff
@@ -22,8 +22,11 @@ jobs:
 
     strategy:
       matrix:
-        os: [macos-14]
-        xcode: [Xcode_15.2, Xcode_16]
+        include:
```

Comment:
array still needed in this one.

Reply:
Are you sure? I see both [`pod_lib_lint (macos-14, Xcode_15.2)`](https://github.com/firebase/firebase-ios-sdk/actions/runs/11748780733/job/32733607729?pr=14061) and [`pod_lib_lint (macos-15, Xcode_16.1)`](https://github.com/firebase/firebase-ios-sdk/actions/runs/11748780733/job/32733608100?pr=14061) in the checks. Is there another one we should run?

Reply:
I see. It works because it only supports iOS.